packages/redux/src/forms/actions/factories/fetchFormSchemaFactory.ts

```diff
@@ -34,9 +41,14 @@ import * as actionTypes from '../../actionTypes';
  *
  * @returns {FetchFormSchemaThunkFactory} Thunk factory.
  */
-export default getFormSchema =>
-  (schemaCode, query, config) =>
-  async dispatch => {
+const getFormSchema =
```

Comment:
This should be called `fetchFormSchemaFactory` and I would give it a type instead of typing the individual constituents. Also, make sure you reuse the same type for the parameters of the factory function as the parameters for the client function as they are the same in this case.